packages/rancher-prom2teams/generated-changes/patch/templates/_helpers.tpl.patch

```diff
@@ -0,0 +1,63 @@
+--- charts-original/templates/_helpers.tpl
++++ charts/templates/_helpers.tpl
+@@ -1,4 +1,31 @@
+ {{/* vim: set filetype=mustache: */}}
++
++{{- define "system_default_registry" -}}
++{{- if .Values.global.cattle.systemDefaultRegistry -}}
++{{- printf "%s/" .Values.global.cattle.systemDefaultRegistry -}}
++{{- end -}}
++{{- end -}}
++
++{{/*
++Windows cluster will add default taint for linux nodes,
++add below linux tolerations to workloads could be scheduled to those linux nodes
++*/}}
++
++{{- define "linux-node-tolerations" -}}
++- key: "cattle.io/os"
++  value: "linux"
++  effect: "NoSchedule"
++  operator: "Equal"
++{{- end -}}
++
++{{- define "linux-node-selector" -}}
++{{- if semverCompare "<1.14-0" .Capabilities.KubeVersion.GitVersion -}}
++beta.kubernetes.io/os: linux
++{{- else -}}
++kubernetes.io/os: linux
++{{- end -}}
++{{- end -}}
++
+ {{/*
+ Expand the name of the chart.
+ */}}
+@@ -25,6 +52,28 @@
+ {{- end -}}
+ 
+ {{/*
++Allow the release namespace to be overridden for multi-namespace deployments in combined charts
++*/}}
++{{- define "prom2teams.namespace" -}}
++  {{- if .Values.global.namespaceOverride -}}
++    {{- .Values.global.namespaceOverride -}}
++  {{- else -}}
++    {{- .Release.Namespace -}}
++  {{- end -}}
++{{- end -}}
++
++{{/*
++Create the name of the service account to use
++*/}}
++{{- define "prom2teams.serviceAccountName" -}}
++{{- if .Values.serviceAccount.create }}
++{{- default (include "prom2teams.fullname" .) .Values.serviceAccount.name }}
++{{- else }}
++{{- default "default" .Values.serviceAccount.name }}
++{{- end }}
++{{- end }}
++
++{{/*
+ Create chart name and version as used by the chart label.
+ */}}
+ {{- define "prom2teams.chart" -}}
```

Comment:
What does this one do ?

Reply:
This template is part of Helm's default template, and it is used by the template `prom2teams.labels`. 

Since it is used in only one place, do you want me to remove it?

Reply:
it was removed